dk.sdu.se4.common/src/main/java/dk/sdu/se4/common/service/GameService.java

```diff
@@ -0,0 +1,32 @@
+package dk.sdu.se4.common.service;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.InputProcessor;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+
+import java.util.List;
+
+public interface GameService {
+
+  void create();
+
+  void addMapService(MapService mapService);
+
+  void removeMapService(MapService mapService);
```

Comment:
Should not be part of the interface

Reply:
no

Reply:
I remove it again

Reply:
but it cant the interface is the contract of the class and it sai with methods shot bee there

Reply:
I remove create and add and remove